### DISCUSSION TITLE
Migrating from vuex-orm and VuexORMLocalForage plugin

### DISCUSSION THREAD

**Discussion by jpmc3630** [vuejs/pinia#1534] [Help and Questions]
We are hoping to migrate to pinia and pinia-orm, but we are currently using the VuexORMLocalForage plugin (with vuex-orm and vuex3). 

I can't find any info about pinia-orm and localforage. Is it possible to use the indexdb instead of local storage with pinia-orm?

**Comment by CodeDredd** [comment]
I think it should be possible. try to use https://github.com/iendeavor/pinia-plugin-persistedstate-2.
I made you an simple example:
https://codesandbox.io/s/wispy-shape-516trq?file=/src/App.vue
